pages/common/gcloud.md

```diff
@@ -0,0 +1,32 @@
+# gcloud
+
+> The official CLI tool for Google Cloud Platform.
+> You can use this tool to perform common platform tasks.
+
+- List all properties in your active configuration. Including: current authorized account, current project, and the default region and zone, if set:
```

Comment:
The second line can be dropped. Also, we prefer to use a formal tone in our pages, so let's change your => one's.

Reply:
SGTM